configure

```diff
@@ -13526,12 +13522,6 @@ case $host in #(
     arch=i386; system=beos ;; #(
   i[3456]86-*-cygwin) :
     arch=i386; system=cygwin ;; #(
-  i[3456]86-*-darwin*) :
```

Comment:
Given the `if $arch64` below, it looks like it is / was possible to have a 64-bit x86 MacOS system reported as `i[3456]86-*-darwin`.  Are we sure this is no longer the case?

Reply:
I'm sure enough.  I did some investigation on the web and also looked at our `config.guess` file.  It seems to me that it is unlikely we would obtain a target triplet that didn't start `x86_64-` on any vaguely recent macOS system.  If anyone is trying to cross compile to iOS, they will presumably have to specify the target triplet manually in any case, and as far as I understand it the `x86_64-` one has been canonical for some time.